.travis.yml

```diff
@@ -35,3 +41,7 @@ script: vendor/bin/phpunit --coverage-clover=coverage.clover
 after_script:
   - wget https://scrutinizer-ci.com/ocular.phar
   - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
```

Comment:
this should be skipped on PHP nightly, which cannot generate code coverage (XDebug does not support PHP 7 yet).
If you don't skip the call and the PHP 7 jobs is the first one to finish, Ocular will notify Scrutinizer that the code coverage is missing and that it should stop waiting.